session/src/test/java/uk/gov/di/ipv/cri/common/api/handler/SessionHandlerTest.java

```diff
@@ -69,7 +69,11 @@ void shouldCreateAndSaveAddressSession()
 
         UUID sessionId = UUID.randomUUID();
         SharedClaims sharedClaims = new SharedClaims();
-        Map<String, String> requestHeaders = Map.of("header-name", "headerValue");
+        Map<String, String> requestHeaders =
+                Map.of(
+                        "header-name", "headerValue",
+                        "x-forwarded-for", "192.0.2.0");
+
         String subject = "subject";
         String persistentSessionId = "persistent_session_id_value";
         String clientSessionId = "govuk_signin_journey_id_value";
```

Comment:
can you verify that the sessionRequest has been set with the client ip address please? i.e.
`verify(sessionRequest).setClientIpAddress("the ip address");`